cluster/libvirt-coreos/util.sh

```diff
@@ -156,6 +158,32 @@ function initialize-network {
   virsh net-create "$ROOT/network_kubernetes_pods.xml"
 }
 
+function destroy-discovery {
+  if [ -d $ROOT/discovery.etcd ]; then
+    set +e
+    kill $(ps a | grep -v grep | grep http://192.168.10.200:4001 | awk '{print $1}')
```

Comment:
this IP address seems like magic.  Where did it come from, at least use a variable.